core/client/fs/src/main/java/alluxio/client/file/options/CreateDirectoryOptions.java

```diff
@@ -225,7 +224,7 @@ public CreateDirectoryTOptions toThrift() {
     options.setAllowExists(mAllowExists);
     options.setRecursive(mRecursive);
     options.setTtl(mTtl);
-    options.setTtlAction(ThriftUtils.toThrift(mTtlAction));
+    options.setTtlAction(TtlAction.toThrift(mTtlAction));
```

Comment:
Why are some of the `toThrift` calls static methods and others instance methods?

Reply:
They are all instance methods except for `TtlAction.toThrift`. This is because we have special handling for `null` `TtlAction` where we turn it into `TtlAction.DELETE`